src/pokemon.c

```diff
@@ -1162,6 +1163,8 @@ void CreateBoxMon(struct BoxPokemon *boxMon, u16 species, u8 level, u8 fixedIV,
                 totalRerolls += 1;
             if (I_FISHING_CHAIN && gIsFishingEncounter)
                 totalRerolls += CalculateChainFishingShinyRolls();
+            if (gDexnavBattle)
```

Comment:
There are still some inconsistencies about ``DexNav`` vs ``Dexnav`` capitalisation when it comes to functions and variable names